frontend/api/index.js

```diff
@@ -113,13 +113,17 @@ app.get('/config/project-overrides', (req, res) => {
       value: envToBool('DISABLE_INVITE_LINKS', false),
     },
     { name: 'albacross', value: process.env.ALBACROSS_CLIENT_ID },
-    {name: 'useSecureCookies', value: envToBool('USE_SECURE_COOKIES', true)},
-    {name: 'cookieSameSite', value: process.env.USE_SECURE_COOKIES}
+    { name: 'useSecureCookies', value: envToBool('USE_SECURE_COOKIES', true) },
+    { name: 'cookieSameSite', value: process.env.USE_SECURE_COOKIES },
+    {
+      name: 'githubApURL',
```

Comment:
```suggestion
      name: 'githubAppURL',
```

Reply:
Am I right in saving there's a Python equivelent of this file which needs to be updated @matthewelwell ?

Reply:
Added